keyboards/idobo/keymaps/drewdobo/keymap.c

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2020 Drew Smathers
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+// Keyboard Layers
+#define _QW 0
```

Comment:
I'd recommend using an enum to manage your layers here in place of Lines 20-23:

```c

enum layer_names {
    _QW,
    _QW_MAC,
    _FN,
    _MOUSE,
};
```

Doing it this way means adding/deleting/reordering only requires editing lines in the whole, as opposed to editing a bunch of hard definitions.

Reply:
Ah nice, updated to use enum instead.